src/uncrustify.cpp

```diff
@@ -479,6 +480,36 @@ int main(int argc, char *argv[])
       }
    }
 
+   /* Set config options using command line arguments.*/
+   idx = 0;
+   while ((p_arg = arg.Params("--set", idx)) != NULL)
+   {
+      char buffer[256];
+      strcpy(buffer, p_arg);
```

Comment:
Ok. I'll try like the safe version under Linux, OSX. I don't find anything in the autoconf file but I see some C99 references in the build/source files.